frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -32,9 +32,10 @@ describe("scenarios > question > new", () => {
       openOrdersTable();
       cy.wait("@dataset");
       cy.url().should("include", "question#");
-      cy.get("[data-metabase-event$='Go to Notebook Mode']").click();
+      // Isolate icons within "QueryBuilder" scope because there is also `.Icon-sql` in top navigation
```

Comment:
@paulrosenzweig - this is why targetting CSS is tricky. However, I believe that scoping those icons within "QueryBuilder" should do the job.

Reply:
Yeah... typically icons are unique on a page, but unfortunately not here. I like using `.QueryBuilder` for scoping. If we end up using that a lot we could add a helper `queryBuilder()` like we have for `modal()`, `sidebar()`, etc.